code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -594,8 +594,8 @@
 	will pulse whatever's connected to it. Pulsing the first activation pin will send a message. Message \
 	can be send to multiple recepients. Addresses must be separated with ; symbol."
 	icon_state = "signal"
-	complexity = 4
-	cooldown_per_use = 5
+	complexity = 2
```

Comment:
also buffed the normal ntnet component which only allows the 3 arg format. lowered cooldown and halved complexity.